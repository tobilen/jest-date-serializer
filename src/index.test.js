const serializer = require('../dist');

describe('jest-date-serializer', () => {
  it('serializes properly', () => {
    expect(new Date(2018, 6, 1, 0, 0, 0, 0)).toMatchSnapshot();
  });

  it('returns stripped date', () => {
    const utcDate = new Date(Date.UTC(2018, 6, 1, 0, 0, 0, 0));
    const cestDate = new Date(
      utcDate.getTime() - 120 * 60000,
    );
    expect(serializer.print(cestDate)).toBe(utcDate.toUTCString());
  })
});
