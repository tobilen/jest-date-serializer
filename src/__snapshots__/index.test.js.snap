// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-date-serializer serializes properly 1`] = `Sun, 01 Jul 2018 00:00:00 GMT`;
